preppal-fe/src/pages/home.tsx

```diff
@@ -1,5 +1,98 @@
+    title: 'Grilled Cheese Sandwich',
+    imageUrl: 'https://images.media-allrecipes.com/userphotos/2206436.jpg',
+    description: 'Learn how to make a grilled cheese sandwich in a nonstick pan with buttered bread and American Cheddar for a classic hot sandwich.',
+    link: 'https://www.allrecipes.com/recipe/23891/grilled-cheese-sandwich/',
+  },
+  {
+    id: 4,
+    title: 'Simple Macaroni and Cheese',
+    imageUrl: 'https://www.the-girl-who-ate-everything.com/wp-content/uploads/2021/07/stovetop-mac-n-cheese-10.jpg',
+    description: 'Quick, easy, and tasty macaroni and cheese dish. Fancy, designer mac and cheese often costs forty or fifty dollars to prepare when you have so many expensive cheeses, but they aren\'t always the best tasting. This simple recipe is cheap and tasty.',
+    link: 'https://www.allrecipes.com/recipe/238691/simple-macaroni-and-cheese/',
+  },
+  {
+    id: 5,
+    title: 'Grilled Cheese Sandwich',
+    imageUrl: 'https://images.media-allrecipes.com/userphotos/2206436.jpg',
+    description: 'Learn how to make a grilled cheese sandwich in a nonstick pan with buttered bread and American Cheddar for a classic hot sandwich.',
+    link: 'https://www.allrecipes.com/recipe/23891/grilled-cheese-sandwich/',
+  },
+  {
+    id: 6,
+    title: 'Simple Macaroni and Cheese',
+    imageUrl: 'https://www.the-girl-who-ate-everything.com/wp-content/uploads/2021/07/stovetop-mac-n-cheese-10.jpg',
+    description: 'Quick, easy, and tasty macaroni and cheese dish. Fancy, designer mac and cheese often costs forty or fifty dollars to prepare when you have so many expensive cheeses, but they aren\'t always the best tasting. This simple recipe is cheap and tasty.',
+    link: 'https://www.allrecipes.com/recipe/238691/simple-macaroni-and-cheese/',
+  },
+  {
+    id: 7,
+    title: 'Grilled Cheese Sandwich',
+    imageUrl: 'https://images.media-allrecipes.com/userphotos/2206436.jpg',
+    description: 'Learn how to make a grilled cheese sandwich in a nonstick pan with buttered bread and American Cheddar for a classic hot sandwich.',
+    link: 'https://www.allrecipes.com/recipe/23891/grilled-cheese-sandwich/',
+  },
+  {
+    id: 8,
+    title: 'Simple Macaroni and Cheese',
+    imageUrl: 'https://www.the-girl-who-ate-everything.com/wp-content/uploads/2021/07/stovetop-mac-n-cheese-10.jpg',
+    description: 'Quick, easy, and tasty macaroni and cheese dish. Fancy, designer mac and cheese often costs forty or fifty dollars to prepare when you have so many expensive cheeses, but they aren\'t always the best tasting. This simple recipe is cheap and tasty.',
+    link: 'https://www.allrecipes.com/recipe/238691/simple-macaroni-and-cheese/',
+  },
+];
+
+const ExplorePage: React.FC = () => {
+  return (
+    <div className="explore-page">
+      <h1 style={{margin:'20px'}}>Explore</h1>
+      <Row xs="auto" md="auto" lg="auto">
+        {recipes.map((recipe) => (
+          <Col key={recipe.id}>
+            <Card className="recipe-card">
```

Comment:
Is there a way we can refactor this so that when we open our collections page, the cards will be formatted in the same way

Reply:
I'm not sure what you mean here. I separated the card creation into its own function. Do you want to have a separate file to put common functions like that one that we can import and use throughout the website? Because that would probably be a good idea but I don't know if that's what you meant